src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#"
+      method="post" 
+      class="form"
+      data-qa="big"
+    >
+      <label>
+        <img 
+        src="images/Search.svg" 
+        alt="Search" 
+        class="icon"
+        >
```

Comment:
```suggestion
        <img 
          src="images/Search.svg" 
          alt="Search" 
          class="icon"
        >
```
indents (everywhere) 😉